modules/luci-base/luasrc/dispatcher.lua

```diff
@@ -418,6 +418,13 @@ function dispatch(request)
 		ctx.authsession = sid
 		ctx.authtoken = sdat.token
 		ctx.authuser = sdat.username
+
+		if http.getenv("HTTPS") == "on" then
+			local forcehsts = conf.main.forcehsts
+			if forcehsts then
+				http.header("Strict-Transport-Security", 'max-age=%s' % forcehsts)
+			end
+		end
```

Comment:
This change should better go to uhttpd2

Reply:
@yousong  i do not know uhttpd2?

Reply:
Uhttpd2 is the default HTTP server in OpenWrt.  Since the certs, keys etc. required for https are configured on uhttpd, the hsts knob naturally should also go there.

[1] http://lxr.mein.io/source/uhttpd2/proc.c#L150

Reply:
@yousong sorry i am using uhttp and this is the default http server for luci.  Could you point me to the right place where i could add the hsts feature? So i could ads a new PR and get this fixed?
Thanks